Geometry/HGCalGeometry/src/FastTimeGeometryLoader.cc

```diff
@@ -10,123 +10,94 @@
+FastTimeGeometryLoader::FastTimeGeometryLoader() {}
+FastTimeGeometryLoader::~FastTimeGeometryLoader() {}
 
+FastTimeGeometry* FastTimeGeometryLoader::build(const FastTimeTopology& topology) {
   // allocate geometry
-  FastTimeGeometry* geom = new FastTimeGeometry (topology);
-  unsigned int numberOfCells = topology.totalGeomModules(); // both sides
-  int                detType = topology.detectorType();
+  FastTimeGeometry* geom = new FastTimeGeometry(topology);
+  unsigned int numberOfCells = topology.totalGeomModules();  // both sides
+  int detType = topology.detectorType();
 #ifdef EDM_ML_DEBUG
-  std::cout << "Number of Cells " << numberOfCells <<  " for type " << detType
-	    << " of sub-detector " << topology.subDetector() 
-	    << " Shape parameters " << FastTimeGeometry::k_NumberOfShapes
-	    << ":"  << FastTimeGeometry::k_NumberOfParametersPerShape 
-	    << std::endl;
+  std::cout << "Number of Cells " << numberOfCells << " for type " << detType << " of sub-detector "
+            << topology.subDetector() << " Shape parameters " << FastTimeGeometry::k_NumberOfShapes << ":"
+            << FastTimeGeometry::k_NumberOfParametersPerShape << std::endl;
 #endif
-  geom->allocateCorners( numberOfCells ) ;
-  geom->allocatePar(FastTimeGeometry::k_NumberOfShapes,
-		    FastTimeGeometry::k_NumberOfParametersPerShape);
+  geom->allocateCorners(numberOfCells);
+  geom->allocatePar(FastTimeGeometry::k_NumberOfShapes, FastTimeGeometry::k_NumberOfParametersPerShape);
 
   // loop over modules
-  ParmVec params(FastTimeGeometry::k_NumberOfParametersPerShape,0);
+  ParmVec params(FastTimeGeometry::k_NumberOfParametersPerShape, 0);
   unsigned int counter(0);
 #ifdef EDM_ML_DEBUG
-  std::cout << "FastTimeGeometryLoader with # of transformation matrices " 
-	    << numberOfCells << std::endl;
+  std::cout << "FastTimeGeometryLoader with # of transformation matrices " << numberOfCells << std::endl;
 #endif
-  for (unsigned itr=0; itr<numberOfCells; ++itr) {
-    int zside  = (itr == 0) ? 1 : -1;
+  for (unsigned itr = 0; itr < numberOfCells; ++itr) {
+    int zside = (itr == 0) ? 1 : -1;
 #ifdef EDM_ML_DEBUG
-    std::cout << "FastTimeGeometryLoader:: Z:Layer:Type " << zside
-	      << ":" << detType <<std::endl;
+    std::cout << "FastTimeGeometryLoader:: Z:Layer:Type " << zside << ":" << detType << std::endl;
 #endif
-    double zv = zside*(topology.dddConstants().getZPos(detType));
-    const CLHEP::HepRep3x3 rotation = (zside > 0) ?
-      CLHEP::HepRep3x3(1,0,0,0,1,0,0,0,1) :
-      CLHEP::HepRep3x3(-1,0,0,0,1,0,0,0,-1);
-    const CLHEP::HepRotation hr ( rotation );
-    const CLHEP::Hep3Vector h3v(0,0,zv);
-    const HepGeom::Transform3D ht3d (hr, h3v);
-    DetId detId = (DetId)(FastTimeDetId(detType,0,0,zside));
+    double zv = zside * (topology.dddConstants().getZPos(detType));
+    const CLHEP::HepRep3x3 rotation =
+        (zside > 0) ? CLHEP::HepRep3x3(1, 0, 0, 0, 1, 0, 0, 0, 1) : CLHEP::HepRep3x3(-1, 0, 0, 0, 1, 0, 0, 0, -1);
```

Comment:
@makortel and @smuzaffar - it seems to me that this line is a bit long. I would keep the options in two lines - they would be easier to compare.